sql/core/src/test/resources/sql-tests/inputs/postgreSQL/interval.sql

```diff
@@ -342,3 +333,10 @@ SELECT interval '1 2:03:04' minute to second;
 -- select make_interval(secs := 'inf');
 -- select make_interval(secs := 'NaN');
 -- select make_interval(secs := 7e12);
+
+-- test justify_hours() and justify_days()
+SELECT justify_hours(interval '6 months 3 days 52 hours 3 minutes 2 seconds') as `6 mons 5 days 4 hours 3 mins 2 seconds`;
+SELECT justify_days(interval '6 months 36 days 5 hours 4 minutes 3 seconds') as `7 mons 6 days 5 hours 4 mins 3 seconds`;
+
+-- test justify_interval()
+SELECT justify_interval(interval '1 month -1 hour') as `1 month -1 hour`;
```

Comment:
Can you move these tests into the original position? I just want to keep the diff between this test file and the Pg original test smaller.

Reply:
OK, I just wanted to make the diff smalle. I'd move them back